### PR TITLE
James 2156

### PR DESCRIPTION
**Pull request by P3trur0**
This solves #999.

### CODE REVIEW

File: mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/openjpa/AbstractJPAMailboxMessage.java [hunk 1]
```diff
@@ -64,7 +41,17 @@
 import org.apache.openjpa.persistence.jdbc.ElementJoinColumns;
 import org.apache.openjpa.persistence.jdbc.Index;
 
-import com.google.common.base.Objects;
+import javax.mail.Flags;
+import javax.persistence.*;
```
Comment (chibenwa) [inline mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/openjpa/AbstractJPAMailboxMessage.java]: We do not use wildcard imports

File: mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageWithAttachmentMapperTest.java [hunk 2]
```diff
@@ -33,7 +33,6 @@
 import org.apache.james.mailbox.exception.MailboxException;
```
Comment (chibenwa) [inline mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageWithAttachmentMapperTest.java]: If possible, I would like to do no changes to this test suite.

**Comment by openpaas-james** [comment]
Build triggered. sha1 is original commit.

**Comment by openpaas-james** [comment]
Build started sha1 is original commit.

**Pull request closed without merging**